public/js/discover/searchList.js

```diff
@@ -83,15 +83,10 @@ export function selectCard(id = null, scroll = false) {
  * @return {string} The formatted phone number.
  */
 function formatPhoneNumber(number) {
```

Comment:
Does the input need to be sanitized/do we know for sure that it matches ```[0-9]+```?

Reply:
Yes, the form only accepts numbers.

Reply:
Cool, thanks for the clarification.